tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -6553,23 +6553,39 @@ Status ConvertCombinedNMS(OpConverterParams* params) {
 Status ConvertResize(OpConverterParams* params) {
   const auto& inputs = params->inputs;
   const auto& node_def = params->node_def;
-  TF_RETURN_IF_ERROR(
-      CheckInputsWeights(*params, {{"input", false}, {"size", true}}));
+  TF_RETURN_IF_ERROR(CheckInputsWeights(*params,
+      {{"input", TrtInputArg::kTensor}, {"size", TrtInputArg::kBoth}}));
   TF_RETURN_IF_ERROR(AllowDataTypes(
       *params, {DataType::DT_FLOAT, DataType::DT_HALF, DataType::DT_INT32}));
 
-  // Get input tensor. Transpose it from NHWC to NCHW.
+  // Get input tensor.
   ITensorProxyPtr inputs_tensor = inputs.at(0).tensor();
-
   TFTRT_RETURN_ERROR_IF_NULLPTR(inputs_tensor, params->node_def.name());
-
-  // Get output size. It must constain two values i.e. [H_out, W_out]
-  TRT_ShapedWeights weights = inputs.at(1).weights();
-  if (weights.count() != 2) {
-    return errors::Unimplemented("Resize to shape=[] is not supported, at ",
-                                 node_def.name());
+  const auto inputs_dims = inputs_tensor->getDimensions();
+  const bool static_input_shape = HasStaticShape(inputs_dims);
```

Comment:
NIT: we can replace the use of static_input_shape && const_output_size with
 const bool static_output_shape = HasStaticShape(inputs_dims) && const_output_size

the put this new variable closer to where it is used.

Reply:
Yes, this might be a bit cleaner indeed. Updated it as per your suggestion.